config.go

```diff
@@ -208,6 +213,7 @@ func DefaultConfig() *Config {
 		SnapshotInterval:   120 * time.Second,
 		SnapshotThreshold:  8192,
 		LeaderLeaseTimeout: 500 * time.Millisecond,
+		LogLevel:           "TRACE",
```

Comment:
Do we want TRACE to be the default level? I suspect INFO is more reasonable, and at very least since we don't have any messages below debug, we should set it to debug.

Reply:
I totally agree. When I first set it to TRACE I was thinking about preserving every old logger output. As you said, since we don't actually have any messages below debug, it's more reasonable setting it to DEBUG.